plotlykt-core/src/jvmMain/kotlin/scientifik/plotly/FileExport.kt

```diff
@@ -45,36 +65,53 @@ fun Plot2D.makeHtml(): String {
  * Create a standalone html with the plot
  * @param file the reference to html file. If null, create a temporary file
  * @param show if true, start the browser after file is created
+ * @param provideAssets use this to control the way assets (js-files/images/etc)
+ *                      will be referenced. Check [specific entries][AssetsProvidingType]
+ *                      for details
  */
-fun Plot2D.makeFile(file: File? = null, show: Boolean = true) {
+fun Plot2D.makeFile(
+    file: File? = null,
+    show: Boolean = true,
+    provideAssets: AssetsProvidingMode = AssetsProvidingMode.Online
```

Comment:
I suggest to add a nullable `assetPath: String? = null` here to point to existing asset directory. If null, use cdn.